site/docs/configuration.md

```diff
@@ -38,7 +38,7 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.parquet.row-group-size-bytes | 134217728 (128 MB) | Parquet row group size                             |
 | write.parquet.page-size-bytes      | 1048576 (1 MB)     | Parquet page size                                  |
 | write.parquet.dict-size-bytes      | 2097152 (2 MB)     | Parquet dictionary page size                       |
-| write.parquet.compression-codec    | gzip               | Parquet compression codec                          |
+| write.parquet.compression-codec    | gzip               | Parquet compression codec; uncompressed, snappy, gzip, etc. For more options: [CompressionCodecName](https://github.com/apache/parquet-mr/blob/parquet-1.12.x/parquet-common/src/main/java/org/apache/parquet/hadoop/metadata/CompressionCodecName.java) |
```

Comment:
Rather than adding significantly more text to a table, let's make "compression codec" the link text. Also, this can simply list the other codec names and still be smaller than what is here. Including "etc" takes nearly as much space as "zstd" so let's just list these: zstd, brotli, lz4, gzip, snappy, uncompressed.